package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@sinet/pathlint",
+  "version": "1.0.0",
+  "description": "Enforces specific file/directory name patterns for uniformity. It matches files/directories in the current working directory. Great for projects involving several developers.",
+  "author": {
+    "name": "Vincent John Cagay",
```

Comment:
Should we have the author be `School Improvement Network` to be consistent with the other open source projects we've released, e.g. https://github.com/School-Improvement-Network/lapin/blob/master/package.json#L16